src/darwin/CHIPTool/CHIPTool/View Controllers/Temperature Sensor/TemperatureSensorViewController.m

```diff
@@ -123,7 +123,7 @@ - (void)setupUI
     _deltaInFahrenheitTextField = [UITextField new];
     _deltaInFahrenheitTextField.keyboardType = UIKeyboardTypeNumberPad;
     UILabel * deltaInFahrenheitLabel = [UILabel new];
-    [deltaInFahrenheitLabel setText:@"Delta (F):"];
+    [deltaInFahrenheitLabel setText:@"Delta (°C):"];
```

Comment:
please rename this deltaInCelsiusLabel and deltaCelsiusView etc.

Otherwise later people will wonder why we place C on something named 'Fahrenheit'